test/cypress/integration/request-psp-test-account/submit-request-for-psp-test-account.cy.js

```diff
@@ -36,29 +41,11 @@ describe('Request PSP test account: submit request', () => {
       setupStubs('NOT_STARTED', 'NOT_STARTED')
       cy.visit(requestStripeTestAccountUrl)
 
-      cy.get('button').contains('Submit request').click()
+      cy.get('button').contains('Get a Stripe test account').click()
 
-      cy.get('h1').should('contain', 'Thanks for requesting a Stripe test account')
-    })
-  })
-
-  describe('PSP test account stage is REQUEST_SUBMITTED', () => {
-    it('should show "Account already requested" page', () => {
-      setupStubs('NOT_STARTED', 'REQUEST_SUBMITTED')
-      cy.visit(requestStripeTestAccountUrl)
-
-      cy.get('button').contains('Submit request').click()
-      cy.get('h1').should('contain', 'Account already requested')
-    })
-  })
-
-  describe('PSP test account stage is CREATED', () => {
-    it('should show "Stripe test account already set up" page', () => {
-      setupStubs('NOT_STARTED', 'CREATED')
-      cy.visit(requestStripeTestAccountUrl)
-
-      cy.get('button').contains('Submit request').click()
-      cy.get('h1').should('contain', 'Stripe test account already set up')
```

Comment:
Since requesting a new Stripe test account is synchronous, these tests can be deleted as it's no longer possible to try and request a new Stripe account once it's been created.